### PR TITLE
Fix: support more UTI types for movie and audio 

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

Add support of `quicktime` as movie type for `conformsToMovieType` method.
Add support of `m4a` and `mp4` as audio type for `conformsToMovieType` method.

### CODE REVIEW

File: Source/UTIHelper.swift [hunk 1]
```diff
@@ -133,23 +137,32 @@ public final class UTIHelper: NSObject {
...
-        
+        guard let uti = convertToUti(mime: mime) else {
+            return false
+        }
         if #available(iOS 14, *) {
-            return conformsTo(uti: uti, type: .audio)
+            let audioTypes: [UTType] = [.audio, .mpeg4Audio]
+            return audioTypes.first(where: {
+                conformsTo(uti: uti, type: $0)
+            }) != nil
```
Comment (johnxnguyen) [inline Source/UTIHelper.swift:149]: I think you can use `contains` here:

```swift
return audioTypes.contains { conformsTo(uti: uti, type: $0) }
```

File: Source/UTIHelper.swift [hunk 2]
```diff
@@ -133,23 +137,32 @@ public final class UTIHelper: NSObject {
...
+        }
         
         if #available(iOS 14, *) {
-            return conformsTo(uti: uti, type: .movie) ||
-                conformsTo(uti: uti, type: .mpeg4Movie)
+            let movieTypes: [UTType] = [.movie, .mpeg4Movie, .quickTimeMovie]
+            
+            return movieTypes.first(where: {
+                conformsTo(uti: uti, type: $0)
+            }) != nil
```
Comment (johnxnguyen) [inline Source/UTIHelper.swift:165]: Same here.

File: Tests/UTIHelperTests.swift [hunk 3]
```diff
@@ -59,7 +79,7 @@ final class UTIHelperTests: XCTestCase {
 
         suts.forEach { sut in
             // when & then
-            XCTAssert(UTIHelper.conformsToImageType(uti: sut), "\(sut) does not conorms to image type")
+            XCTAssert(UTIHelper.conformsToImageType(uti: sut), "\(sut) does not conforms to image type")
```
Comment (johnxnguyen) [inline Tests/UTIHelperTests.swift:82]:
```suggestion
            XCTAssert(UTIHelper.conformsToImageType(uti: sut), "\(sut) does not conform to image type")
```

File: Tests/UTIHelperTests.swift [hunk 4]
```diff
@@ -50,6 +50,26 @@ final class UTIHelperTests: XCTestCase {
...
+
+    func testThatCommonFilesConformsAudioType() {
+        // given
+        let suts = ["audio/mp4",
+                    "audio/mpeg",
+                    "audio/x-m4a"]
+
+        suts.forEach { sut in
+            // when & then
+            XCTAssert(UTIHelper.conformsToAudioType(mime: sut), "\(sut) does not conforms to audio type")
```
Comment (johnxnguyen) [inline Tests/UTIHelperTests.swift:69]:
```suggestion
            XCTAssert(UTIHelper.conformsToAudioType(mime: sut), "\(sut) does not conform to audio type")
```

**Review by johnxnguyen — APPROVED** [review]

**Review by KaterinaWire — APPROVED** [review]

**Comment by billypchan** [comment]
@zenkins release patch

**Pull request merged**